internal/format/format.go

```diff
@@ -0,0 +1,26 @@
+package format
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hazelcast/hazelcast-go-client/types"
+)
+
+// Fmt defines output format for different SQL types
+func Fmt(v interface{}) string {
+	var formatted string
+	switch t := v.(type) {
+	case types.LocalDate:
+		formatted = time.Time(t).Format("2006-01-02")
```

Comment:
Instead of setting the `formatted` variable and then returning it, you can also directly return the formatted string.

Reply:
Thanks, refactored https://github.com/hazelcast/hazelcast-commandline-client/pull/130/commits/c11ffbd7e83a4dcdb4c01b2b2f9a92e26e7ad17d